File: class23_homework/A-Ivaylo-Markov-12/ruby/people/group.rb
class Group

	def initialize
		@members = []
	end
	
	def get_member id
		return @members[id]
	end
	
	def add_member new_member
		@members << new_member
	end
	
	def get_size
		@members.size
	end
	
	def clear
		@members.clear
	end
	

end

File: class23_homework/A-Ivaylo-Markov-12/java/src/org/elsys/se/class23/homework/people/Group.java
package org.elsys.se.class23.homework.people;
import java.util.*;

public class Group {
	private List<Passenger> members;
	
	public Group() {
		members = new ArrayList<Passenger>();
	}
	
	public Passenger getMember(int i) {
		return members.get(i);
	}
	
	public void addMember(Passenger newMember) {
		members.add(newMember);
	}
	
	public int getSize() {
		return members.size();
	}
	
	public void clear() {
		members.clear();
	}

}
